### PR TITLE
Add config/naming to Radarr

### PR DESCRIPTION
**Pull request by davidnewhall**
Adds `config/naming` endpoints to Radarr. Pretty much copy/paste from Sonarr.

### CODE REVIEW

File: radarr/naming.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+// Naming represents the config/naming endpoint in Radarr.
+type Naming struct {
+	RenameMovies             bool   `json:"renameMovies,omitempty"`
+	ReplaceIllegalCharacters bool   `json:"replaceIllegalCharacters,omitempty"`
+	IncludeQuality           bool   `json:"includeQuality,omitempty"`
+	ReplaceSpaces            bool   `json:"replaceSpaces,omitempty"`
+	ID                       int64  `json:"id"` // ID must always be 1 (Oct 10, 2022)
+	ColonReplacementFormat   string `json:"colonReplacementFormat,omitempty"`
+	StandardMovieFormat      string `json:"standardMovieFormat,omitempty"`
+	MovieFolderFormat        string `json:"movieFolderFormat,omitempty"`
```
Comment (Fuochi) [inline radarr/naming.go:24]: I think these 2 cannot be empty

File: radarr/naming.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+const bpNaming = APIver + "/config/naming"
+
+// Naming represents the config/naming endpoint in Radarr.
+type Naming struct {
+	RenameMovies             bool   `json:"renameMovies,omitempty"`
+	ReplaceIllegalCharacters bool   `json:"replaceIllegalCharacters,omitempty"`
+	IncludeQuality           bool   `json:"includeQuality,omitempty"`
+	ReplaceSpaces            bool   `json:"replaceSpaces,omitempty"`
+	ID                       int64  `json:"id"` // ID must always be 1 (Oct 10, 2022)
+	ColonReplacementFormat   string `json:"colonReplacementFormat,omitempty"`
```
Comment (Fuochi) [inline radarr/naming.go:22]: do you want to start using enum here as well? honestly, I think we can avoid it.
for your reference, values are: `delete`, `dash`, `spaceDash`, and `spaceDashSpace`

File: test_methods.go [hunk 3]
```diff
@@ -33,18 +33,20 @@ const (
 )
 
 // GetMockServer is used in all the http tests.
+//
+//nolint:lll
```
Comment (Fuochi) [inline test_methods.go:37]: do you want to directly remove `lll` in all tests file from `.golangci.yml`?

**Review by Fuochi — APPROVED** [review]
couple of comments, but not merge blockers, I'll try to find time this week to fix my other 2 PRs

File: test_methods.go (hunk 3 above)
Comment (davidnewhall) [reply test_methods.go:37]: This isn't a test file, but yes, I do want to add this file as part of the test-file regex in `.golangci.yml`. However, I want to make a new package `starrtest` to put this in first. That requires touching every `_test.go` file in this library, so I was going to do it in another PR (after you get all yours merged). I started to do it in this one, until I realized how many things need to be changed. For now I'm okay leaving this in here; I'll re-work it later.

File: radarr/naming.go (hunk 2 above)
Comment (davidnewhall) [reply radarr/naming.go:22]: I like the idea of enums, but I'm not married to the idea of putting on everything.

File: radarr/naming.go (hunk 1 above)
Comment (davidnewhall) [reply radarr/naming.go:24]: Good point, I only tested with the weird `id` member. I'll do some more testing with smaller payloads.

Comment (davidnewhall) [reply radarr/naming.go]: Indeed they are required.

**Pull request merged**